index.html

```diff
@@ -1,108 +1,258 @@
+            <a href="https://learning.pulsezest.com/courses/php-master-with-sql-in-7-days/" class="btn btn-primary">Buy
+              Now</a>
           </div>
-          <div class="col">
-            <div class="card" style="width: 18rem;">
-                <img src="images/git.png" class="card-img-top" alt="...">
-                <div class="card-body">
-                  <h5 class="card-title">Lean GitHub Basic</h5>
-                  <p class="card-text">Learn gitHub basic alana falana. Performing all the CRUD Opetion with Real and Cloud Storage.</p>
-                  <a href="https://learning.pulsezest.com/courses/learning-firebase/" class="btn btn-primary">By now</a>
-                </div>
-              </div>
+        </div>
+      </div>
+      <div class="col">
+        <!-- Card 4: Lean GitHub Basics -->
+        <div class="card fade-in" style="width: 18rem;">
+          <img src="images/git.png" class="card-img-top" alt="...">
+          <div class="card-body">
+            <h5 class="card-title">Lean GitHub Basics</h5>
+            <p class="card-text">Master GitHub basics and enhance your coding workflow.</p>
+            <a href="https://learning.pulsezest.com/courses/learning-github-basics/" class="btn btn-primary">Buy Now</a>
+          </div>
+        </div>
+      </div>
+      <div class="col">
+        <!-- Card 5: Learn React -->
+        <div class="card fade-in" style="width: 18rem;">
+          <img src="images/react.png" class="card-img-top" alt="...">
+          <div class="card-body">
+            <h5 class="card-title">Learn React</h5>
+            <p class="card-text">Master React and build powerful web applications.</p>
+            <a href="https://learning.pulsezest.com/courses/learn-react/" class="btn btn-primary">Buy Now</a>
           </div>
+
         </div>
       </div>
+    </div>
+  </div>
+
+
+  <!-- Scroll-to-top button -->
+  <button id="scrollToTopBtn" onclick="scrollToTop()" title="Go to top">Top</button>
+  <footer class="footer" style="background-color: #5c2d81; color: #ffffff;">
+    <div class="container py-5">
+        <div class="row">
+            <div class="col-md-4">
+                <h5>About Us</h5>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed vitae fermentum libero.</p>
+            </div>
+            <div class="col-md-4">
+                <h5>Quick Links</h5>
+                <ul class="list-unstyled">
+                    <li><a href="#" style="color: #ffffff;">Home</a></li>
+                    <li><a href="#" style="color: #ffffff;">About</a></li>
+                    <li><a href="#" style="color: #ffffff;">Services</a></li>
+                    <li><a href="#" style="color: #ffffff;">Contact</a></li>
+                </ul>
+            </div>
+            <div class="col-md-4">
+                <h5>Contact Us</h5>
+                <ul class="list-unstyled">
+                    <li>Email: example@example.com</li>

```

Comment:
change email to contact@pulsezest.com

Reply:
then sir give all details like about us, contact detail etc